docs/src/tutorials/project_1.md

```diff
@@ -0,0 +1,473 @@
+    sethue(color)
+    setdash(edge)
+    circle(p, radius, action)
+    return Transformation(p, 0.0)
+end
+```
+
+Placing this inside of the `javis` function, we can draw our head:
+
+```julia
+...
+    Action(:same, :head, (args...) -> circ(O, "black", :stroke, 170)),
+...
+```
+
+![](assets/head.gif)
+
+Now we are getting _ahead_!
+
+> **NOTE:** The ellipses, `...`, in the code block to produce the head represents putting the action inside of the `javis` function.
+> If you are confused about what I mean, please refer to a prior tutorial or the docstring of `javis` in your Julia REPL. 😄
+
+# Placing the Electrodes
+
+To draw our electrodes, it would be useful to have a frame of reference for where the electrodes are supposed to go.
+Let's draw some axes for our electrode locations!
+
+We will need to define a new function that allows us to draw lines.
+This is accomplished by the following function definition:
+
+```julia
+function draw_line(p1 = O, p2 = O, color = "black", action = :stroke, edge = "solid")
+    sethue(color)
+    setdash(edge)
+    line(p1, p2, action)
+end
+```
+
+Now, we can add in some grid lines for our electrode array. 
+The following code places a vertical and horizontal axis as well as an inscribed circle to represent polar placement of the electrodes:
+
+```julia
+...
+        Action(
+            :same,
+            :inside_circle,
+            (args...) -> circ(O, "black", :stroke, 140, "longdashed"),
+        ),
+        Action(
+            :same,
+            :vert_line,
+            (args...) ->
+                draw_line(Point(0, -170), Point(0, 170), "black", :stroke, "longdashed"),
+        ),
+        Action(
+            :same,
+            :horiz_line,
+            (args...) ->
+                draw_line(Point(-170, 0), Point(170, 0), "black", :stroke, "longdashed"),
+...
+```
+
+![](assets/head_gridlines.gif)
+
+Great!
+Now that we have the gridlines, let's add in our nodes!
+As `javis` has no native support for making circles which hold text, we have to define our own function.
+This is not hard to do and requires drawing two circles per electrodes to be "on top of" the grid lines:
+
+```julia
+function outline_circ(
+    p = O,
+    fill_color = "white",
+    outline_color = "black",
+    action = :fill,
+    radius = 25,
+    circ_text = "",
+)
+    sethue(fill_color)
+    circle(p, radius, :fill)
+    sethue(outline_color)
+    circle(p, radius, :stroke)
+    text(circ_text, p, valign = :middle, halign = :center)
+end
+```
+
+Using `outline_circ` function, the following code block creates each electrode at its proper location.
+Feel free to just copy and paste this code below as it took some time to position the electrodes properly (the writer knows this all too well...):
+
+```julia
+...
+        Action(
+            :same,
+            :cz,
+            (args...) -> outline_circ(O, "white", "black", :fill, radius, "Cz"),
```

Comment:
try to use broadcasting for this as you create a lot of code which does the same. 
Or if broadcasting does not work you might want to have a loop for this

Reply:
I LOVE BROADCASTING!!!!